target/linux/ramips/dts/mt7621_netgear_sercomm_bzv.dtsi

```diff
@@ -57,10 +57,37 @@
 	};
 };
 
-&ethernet {
+&gmac0 {
 	mtd-mac-address = <&factory 0x4>;
```

Comment:
r6700-v2 and r6800 derive from this DTSI and both contain label-mac-device and state-default, where at least the former needs an update to gmac0.